layouter/models.py

```diff
@@ -58,17 +61,62 @@ class ContainerPlugin(CMSPlugin):
         (4, FOUR_COL_MARGIN)
     )
 
+    NO_PARALLAX = 0
+    CSS_PARALLAX = 1
+    TRUE_PARALLAX = 2
+    BACKGROUND_IMAGE_PARALLAX_CHOICES = (
+        (NO_PARALLAX, _('No Effect')),
```

Comment:
Maybe we can find a better name for it? Something like "fixed background image" - and "parallax background image".